easy-ui-react/src/Button/Button.module.scss

```diff
@@ -0,0 +1,119 @@
+@import "../styles/common";
+@import "../styles/common-button";
+
+.Button {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--ezui-shape-border-radius-base);
+  cursor: pointer;
+
+  &:focus {
+    box-shadow: var(--ezui-shadow-button);
+    outline: none;
+  }
+
+  &:disabled {
+    pointer-events: none;
+  }
+}
+
+.colorPrimary {
+  @include button-primary-color-aliases();
+}
+
+.colorSecondary {
+  @include button-secondary-color-aliases();
+}
+
+.colorSuccess {
+  @include button-success-color-aliases();
+}
+
+.colorWarning {
+  @include button-warning-color-aliases();
+}
+
+.colorNeutral {
+  @include button-neutral-color-aliases();
+}
+
+.colorSupport {
+  @include button-support-color-aliases();
+}
+
+.colorInverse {
+  @include button-inverse-color-aliases();
+}
+
+.variantFilled {
+  @include button-filled-variant-styles();
+}
+
+.variantOutlined {
+  @include button-outlined-variant-styles();
+}
+
+.variantLink {
+  color: var(--ezui-c-button-resting-color);
+  background-color: transparent;
+  border: var(--ezui-shape-border-width-1) solid transparent;
+  text-decoration: underline;
+
+  &:focus {
+    border: var(--ezui-shape-border-width-1) solid
+      var(--ezui-c-button-hover-focus-color);
+  }
+
+  &:disabled {
+    color: var(--ezui-color-gray-300);
+  }
+
+  &:active:not(:disabled) {
+    color: var(--ezui-c-button-active-color);
+  }
+
+  &:hover:not(:disabled):not(:active) {
+    color: var(--ezui-c-button-hover-focus-color);
+  }
+}
+
+.sizeMd {
+  @include font-style("button");
```

Comment:
checked with design and we do indeed want 15px for the button text, removed changes to font